lib/ansible/modules/cloud/spotinst/spotinst_aws_elastigroup.py

```diff
@@ -0,0 +1,1511 @@
+#!/usr/bin/python
+# Copyright (c) 2017 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+DOCUMENTATION = """
+---
+module: spotinst_aws_elastigroup
+version_added: 2.5
+short_description: Create, update or delete Spotinst AWS Elastigroups
+author: Spotinst
+description:
+  - Can create, update, or delete Spotinst AWS Elastigroups
+    Launch configuration is part of the elastigroup configuration,
+    so no additional modules are necessary for handling the launch configuration.
+    This module requires that you install the spotinst Python SDK (pip install spotinst)
+    You will have to have a credentials file in this location - <home>/.spotinst/credentials
+    The credentials file must contain a row that looks like this
+    token = <YOUR TOKEN>
+    Full documentation available at https://help.spotinst.com/hc/en-us/articles/115003530285-Ansible-
+options:
```

Comment:
documentation for `credentials_path` is missing.

Reply:
Right. Fixed. Thanks!